intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -533,7 +533,19 @@ \subsection{Check your quota}
 status of your quota in a readable format:
 
 \begin{prompt}
-\shellcmd{show_quota}
+%\shellcmd{show\_quota}%
+VSC_DATA:    used 81MB (0%\%%)  quota 25600MB
+VSC_HOME:    used 33MB (1%\%%)  quota 3072MB
+VSC_SCRATCH:   used 28MB (0%\%%)  quota 25600MB
+VSC_SCRATCH_GLOBAL: used 28MB (0%\%%)  quota 25600MB
+VSC_SCRATCH_SITE:   used 28MB (0%\%%)  quota 25600MB
+\end{prompt}
```

Comment:
The show_quota command is something of a problem for the intro-HPC document. I cannot find it on the Leuven and Brussels clusters, and at Antwerp we do it in a slightly different way as in Gent. In fact, we're not married with the command and for me it is even fine to delete it from the document. We show users their quota when they log on to the system.

The question of course is: Is it useful for a non-UAntwerp account to know how show_quota.py works here? It might be to check your quota on scratch at UAntwerp if you log on to UAntwerp, but the other information you get could even be misunderstood. 

I see 3 approaches:
* The UAntwerp version of the tutorial shows the command the way it is at UAntwerp, the UGent version the way it is at UGent and the other instutions can always add.
* We show it to everybody for all universities, with the warning that you will only see the quota for file systems that reside at that location.
* Or we simply omit it from the intro-HPC as both Gent and Antwerp also have a different way of showing the quota to users.